portfolio/src/main/webapp/js/comments.js

```diff
@@ -0,0 +1,150 @@
+var selectedSortType = localStorage.getItem('sortType', selectedSortType);
+
+/**
+ * Adds comments, sorted according to the 'type', to the list '.comments'
+ * @param  {string}  type  type for sorting comments ('date' or 'rating'), if it's unknown type, then 'date' by default
+ */
+function getComments(type) {
+    var params = new URLSearchParams();
+    params.append('type', type);
+    fetch('/comments', {method: 'GET', headers: params}).then(response => response.json()).then((comments) => {
+        const listOfCommentsDOM = document.querySelector('.comments-list');
+        listOfCommentsDOM.innerHTML = '';
+        for (var i = 0; i < comments.length; i++) {
+            listOfCommentsDOM.appendChild(
+                createListElement(comments[i].value.text, comments[i].value.rating, comments[i].value.date, comments[i].key));
```

Comment:
maybe add a variable here to represent `comments[i].value`, since you're reusing it a few times